src/frontend/apps/desk/src/features/teams/components/Member/MemberAction.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button } from '@openfun/cunningham-react';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { DropButton, Text } from '@/components';
+import { Access } from '@/features/teams/api';
+
+import { ModalRole } from './ModalRole';
+
+interface MemberActionProps {
+  teamId: string;
+  access: Access;
+}
+
+export const MemberAction = ({ teamId, access }: MemberActionProps) => {
+  const { t } = useTranslation();
+  const [isModalRoleOpen, setIsModalRoleOpen] = useState(false);
+  const [isDropOpen, setIsDropOpen] = useState(false);
+
+  return (
+    <>
+      <DropButton
+        button={
+          <span
+            aria-label={t('Member options')}
+            className="material-icons"
+            style={{
+              transition: 'all 0.3s ease-in-out',
+              transform: `rotate(${isDropOpen ? '90' : '0'}deg)`,
+            }}
+          >
+            more_vert
+          </span>
+        }
+        onOpenChange={(isOpen) => setIsDropOpen(isOpen)}
+      >
+        <Button
+          onClick={() => setIsModalRoleOpen(true)}
+          color="primary-text"
+          icon={<span className="material-icons">edit</span>}
+        >
+          <Text $theme="primary">{t('Update the role')}</Text>
+        </Button>
```

Comment:
Based on Sam's answer here #87, this button should be disabled if all actions (except get) are false.

Reply:
All the cases should be cover:
- member role
- last owner
- owner with other owner
- admin with owner

Reply:
https://github.com/numerique-gouv/people/pull/73/commits/8267fe3683a10b980d7df4aaf5b02b25f701054d